pkgs/applications/system/booster/default.nix

```diff
@@ -0,0 +1,27 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+}:
+
+buildGoModule rec {
+  pname = "booster";
+  version = "0.9";
+
+  src = fetchFromGitHub {
+    owner = "anatol";
+    repo = pname;
+    rev = version;
+    hash = "sha256-kalVFVBb+ngoUpm+iiIHGS6vBVLEvTVyKuSMSMbp7Qc=";
+  };
+
+  vendorHash = "sha256-GD+nsT4/Y2mTF+ztOC3N560BY5+QSfsPrXZ+dJYtzAw=";
+
+  doCheck = false;
```

Comment:
[Upstream provides tests](https://github.com/anatol/booster/tree/master/tests) so why don't we use them here?

Reply:
the check phase is disabled because integration tests need kernel and other impure resources to function